resources/migrations/000_migrations.yaml

```diff
@@ -4057,3 +4057,14 @@ databaseChangeLog:
               name: read_permissions
               type: text
               remarks: 'Permissions required to view this Card and run its query.'
+  - changeSet:
+      id: 76
+      author: senior
```

Comment:
this is going to get interesting when we try to merge in the stuff from secret project MT since we're using the same author + ID combo there